test/stripe/payment_methods/source_test.exs

```diff
@@ -33,8 +33,7 @@ defmodule Stripe.SourceTest do
 
   describe "detach/2" do
     test "detaches a source from a customer" do
-      assert {:ok, %{deleted: true, id: "src_123"}} =
-               Stripe.Source.detach("src_123", %{customer: "cus_123"})
+      assert {:ok, %{id: "src_123"}} = Stripe.Source.detach("src_123", %{customer: "cus_123"})
```

Comment:
as of [stripe-mock v0.39.0](https://github.com/stripe/stripe-mock/releases/tag/v0.39.0), this endpoint now returns a Stripe.Account object without a `deleted` attribute to match against.

i raised a [ticket on stripe-mock](https://github.com/stripe/stripe-mock/issues/131), since based on Stripe API docs, it seems like this should respond with a Stripe.Source object instead... but regardless, the response will no longer contain a `deleted` attribute either way.

@snewcomer given that these changes are related to updating the stripe-mock version, not voiding an invoice, it seems cleaner to pull the version updates and this test change into a separate PR. let me know what you prefer.